packages/theme-default/src/logic/getSidebarDataGroup.ts

```diff
@@ -1,4 +1,8 @@
-import { pathnameToRouteService, withBase } from '@rspress/runtime';
+import {
+  matchPath as ReactRouterDomMatchPath,
```

Comment:
Prefer to use camelCase for function

```suggestion
  matchPath as reactRouterDomMatchPath,
```